packages/node_modules/@ciscospark/widget-meet/src/components/call-active/index.js

```diff
@@ -19,12 +19,12 @@ export default function ActiveCall({
   onStartSendingVideo,
   onStopSendingAudio,
   onStopSendingVideo,
-  localMediaStreamUrl,
-  remoteMediaStreamUrl,
+  localMediaStream,
+  remoteMediaStream,
   toPersonAvatar,
   toPersonName
 }) {
-  const connectedClass = remoteMediaStreamUrl && styles.callConnected;
```

Comment:
Why are we still storing the remote/local media stream url if we're not using it?